main/webapp/modules/core/langs/translation-en.json

```diff
@@ -674,7 +675,17 @@
         "rows": "rows",
         "records": "records",
         "show": "Show",
-        "hide": "Hide"
+        "hide": "Hide",
+        "text-to-find": "Find:",
+        "replacement-text": "Replace with:",
+        "case-insensitive": "case insensitive",
+        "finding-info1": "Leave blank to add the remplacement string after each character.",
+        "finding-info2": "Check \"regular expression\" to find special characters (new lines, tabulations...) or complex patterns.",
+        "finding-options": "Finding options:",
```

Comment:
I'd remove the "finding-options" and "replacement-options" to save space.  It's self evident already they are options.

Reply:
Also, I would put this under the "Cluster and edit..." on 1st menu.